llmstudio/engine/providers/azure.py

```diff
@@ -433,14 +439,15 @@ def add_tool_instructions(self, tools: list) -> str:
         tool_prompt += """
```

Comment:
should we move the prompts to a "default prompts" library?

Reply:
Yes, I like that suggestion. It's a bit outside the scope of this PR, so I'll open a new one to address it.